packages/pill/src/styles/default.module.css

```diff
@@ -0,0 +1,45 @@
+@import '../../../themes/src/default.css';
+
+/*
+* Colors
+*/
+:root {
+  /* background color */
+  --pill-background-color: var(--color-tarmac-grey-50);
+
+  /* text color */
+  --pill-text-color: var(--color-tarmac-grey-700);
+}
+
+/*
+* Base
+*/
+.pill {
+  font-family: var(--font-family-system);
```

Comment:
I guess it is not needed, due to the mixin `@mixin typography-subheading-3` already contain this rule

Reply:
resolved